polkadot-parachains/encointer-runtime/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = 'encointer-runtime'
 # major.minor revision must match collator node
 # patch revision must match runtime spec_version
-version = '1.3.13'
+version = '1.3.14'
```

Comment:
Same remark as in the node, I think that this should be a minor version upgrade.